### DISCUSSION TITLE
How can I compile this program for Windows myself?

### DISCUSSION THREAD

**Discussion by VanTuz5** [theophanemayaud/video-simili-duplicate-cleaner#96] [General]
How can I compile this program for Windows myself? After all, this is an open source application and it should be possible.

**Comment by theophanemayaud** [comment]
Hi @VanTuz5, I’m not sure if you’re asking nicely or being ironic « after all » 😅

**Comment by VanTuz5** [comment]
English is not my native language, therefore I could formulate incorrectly. I am using an internet translator. I'm sorry if I offended you, I didn't mean it at all.

**Comment by theophanemayaud** [comment]
Oh ok great my bad than ☺️
You should checkout the repository, install qt, open the qt project file in qt, and build.

**Reply by VanTuz5** [reply]
Where can I found Qt?

**Comment by theophanemayaud** [comment]
The ide and framework QT, on QT.io I think, you’ll probably want the free open source community version 😉 they’ll make you go through some many steps before you can get this free version but it’s available 👌

**Reply by VanTuz5** [reply]
I was finally able to return to this program. I tried to open the project file in QT Creator, but the creator said that the version was wrong. What version of QT is needed to compile the program?
And what exactly needs to be done to compile the program?

**Reply by theophanemayaud** [reply]
You can use QT 6.2 or near this version, which is what I'm currently using :) But any minor 6.x version should work

**Reply by theophanemayaud** [reply]
I'm on 6.2.4

**Comment by zhang123-sys** [comment]
![image](https://user-images.githubusercontent.com/63956071/226562730-a0cbe80e-3792-4bc6-906e-7337cffd35d1.png)
Can Qt 5.14.1 (MSVC 2017, 32 bit) work?

**Reply by theophanemayaud** [reply]
Hey ! Since I updated the app to qt 6 I’m not sure it’s retro compatible. But you could either check out a state of the repo before the update to qt 6 (go through the commit history) or simply install qt 6, which is possible to do alongside qt 5 ☺️

**Comment by theophanemayaud** [comment]
I'll soon be trying again to build from windows, although since I only have an arm machine I might encounter additional problems. I'll be sure to update the libraries documentation with my findings !
